app/server/data-loaders/author-page-data.js

```diff
@@ -0,0 +1,23 @@
+import { Author } from "@quintype/framework/server/api-client";
+import get from "lodash/get";
+
+export function loadAuthorPageData(client, authorSlug, config) {
+  let authorDetails = {};
+  const storyFields =
+    "alternative,slug,metadata,story-template,story-content-id,id,headline,hero-image-s3-key,hero-image-metadata,sections,tags,author-name,author-id,authors,created-at,first-published-at,published-at,last-published-at,url";
```

Comment:
move this to constant.js file and use limited field which is being used in cards